CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# 3.0.0
+
+Removes default exports for AbortController. You must now import the `AbortController` object explicitly.This is a breaking change for some users relying on default exports. Upgrading to 3.0 is a one line change:
+
+```js
+// ES Modules Users
+// v2
+import AbortController from "node-abort-controller";
+
+// v3
+import { AbortController } from "node-abort-controller";
+
+// Common JS Users
+// v2
+const AbortController = require("node-abort-controller");
+
+// v3
+const { AbortController } = require("node-abort-controller");
+```
+
 # 2.0.0
 
 - Export AbortController class. This is a non-breaking change for JavaScript users and almost surely a non-breaking change for TypeScript users but we are doing a major version bump to be safe.
```

Comment:
I know this is unrelated, but there is a typo here. This should be talking about `AbortSignal` and not `AbortController`.